src/overview/components/VisitAsListItem.css

```diff
@@ -1,11 +1,11 @@
 .root {
     position: relative;
     display: block;
-    width: 240px;
-    height: 120px;
+    width: 100%;
```

Comment:
any particular reason for this? width is auto by default, which means 100% already for most scenarios.